### DISCUSSION TITLE
Welcome to ani-cli-batch Discussions!

### DISCUSSION THREAD

**Discussion by 75rx** [75rx/ani-cli-batch#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together!

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MaplePlayz** [comment]
Hey hey, i have a quick question, with ani-cli you are able to use ani-cli --dub to watch dubbed anime. Is there anyway to do that with the batch?

**Reply by 75rx** [reply]
It's been a while since I've looked at this code, but no. I don't think you can do that.

However, once you've queued your anime, you can ask ani-cli-batch to give you the download command you can then add whatever arguments you want, to and execute it normally.

Could you open an issue so I could work on this?

**Reply by MaplePlayz** [reply]
yep, i'll do that.
